packages/cli/src/ActiveExecutions.ts

```diff
@@ -190,13 +190,17 @@ export class ActiveExecutions {
 	 * @param {string} executionId The id of the execution to wait for
 	 */
 	async getPostExecutePromise(executionId: string): Promise<IRun | undefined> {
-		// Create the promise which will be resolved when the execution finished
-		const waitPromise = await createDeferredPromise<IRun | undefined>();
-
 		if (this.activeExecutions[executionId] === undefined) {
-			throw new Error(`There is no active execution with id "${executionId}".`);
+			const toThrow = new WorkflowOperationError(
+				`There is no active execution with id "${executionId}".`,
+			);
+			toThrow.severity = 'warning';
+			throw toThrow;
```

Comment:
looking at `WorkflowOperationError` and `SubworkflowOperationError`, I'm not sure if there are any cases so far where the `severity` should not be `warning`.
Maybe we should update those two constructors to include `this.severity = 'warning';`, and replace this with
```suggestion
			throw new WorkflowOperationError(
				`There is no active execution with id "${executionId}".`,
			);
```

Reply:
Good point, changed!